versioned_docs/version-2.0/how_to_guides/evaluation/evaluate_pairwise.mdx

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_label: Run Pairwise Evaluations
+sidebar_position: 8
+---
+
+# Run Pairwise Evaluations
+
+:::note
+Currently, `evaluate_comparative` is only supported in the Python SDK.
+:::
+
+LangSmith supports evaluating existing experiments against each other. This allows you to use an LLM to judge outputs from multiple systems against each other, rather than being confined to evaluating outputs one at a time. Think [Chatbot Arena](https://chat.lmsys.org/) - this is the same concept! To do this, use the `evaluate_comparative` function
```

Comment:
```suggestion
LangSmith supports evaluating existing experiments against each other. This allows you to use automatic evaluators (especially, LLM-based evaluators) to score the outputs from multiple experiments against each other, rather than being confined to evaluating outputs one at a time. Think [Chatbot Arena](https://chat.lmsys.org/) - this is the same concept! To do this, use the `evaluate_comparative` function
```